js/apps/system/_admin/aardvark/APP/react/src/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
As much as I like react, can we remove this logo and instead use ArangoDB?

Reply:
Done.